src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -131,6 +135,25 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
                                 break;
                         }
                     }
+
+                    if (!handled)
+                    {
+                        var baseException = e.GetBaseException();
+
+                        if (baseException is TrustedSignerAlreadyExistsException
```

Comment:
Reason I added this part is to make experience same as `nuget.exe`.
For example if `same trusted signed already exist` then it just show `A trusted signer 'NewtonAuthor333' already exists.`

![image](https://user-images.githubusercontent.com/8766776/115457493-7b16d080-a1d9-11eb-8e59-f1fa0c8cc263.png)

Without this check `dotnet nuget trust` 'll show red marked part assuming it's unhandled.

![image](https://user-images.githubusercontent.com/8766776/115464033-545c9800-a1e1-11eb-8e19-45679e159720.png)

Reply:
I really wish we weren't doing this in the first place. Printing out help doesn't help at all unless the error itself has to do with a command, argument, or option being invalid.

Reply:
Anyway, I don't think we should do this, and we should start a *separate* conversation for not calling `ShowBestHelp()` in every case, and _narrowing_ it down, instead of creating an allowlist like this that definitely does not consistently cover other errors in dotnet.exe nuget commands.

So tl;dr: I see where you're coming from, and it's good that you're aiming to maintain nuget.exe similarity here, but I think "fit in to dotnet.exe" should be higher priority, and also I just don't like this specific approach to it.

Reply:
Ok. Yes, I agree it needs to be addressed separately.
I removed this part and created https://github.com/NuGet/Home/issues/10788 issue to track it.